docs/source/torch.compiler_profiling_torch_compile.md

```diff
@@ -44,11 +41,13 @@ Basics of using torch.profiler and viewing traces
             prof.step()
 
     prof.export_chrome_trace("trace.json")
+```
 
 **Viewing chrome traces**: In the Chrome browser, open chrome://tracing and load the json file. Use the “w” and “s” keys to zoom in and out, and use “a” and “d” to scroll left and right. “?” will show a “help” screen with a list of shortcuts.
 
-.. figure:: _static/img/profiling_torch_compile/basic_chrome_trace.png
-    :alt: Example of a basic chrome trace, visualized in the chrome://tracing viewer
+```{figure}  _static/img/profiling_torch_compile/basic_chrome_trace.png
+    [Alt text] Example of a basic chrome trace, visualized in the chrome://tracing viewer
```

Comment:
I believe you just keep `:alt:`. Ci is failing on [this](https://github.com/pytorch/pytorch/actions/runs/15494205342/job/43628315676?pr=155335#step:11:830) 
```suggestion
:alt: Example of a basic chrome trace, visualized in the chrome://tracing viewer
```